Projects/Android/AndroidManifest.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
 		package="com.drbeef.lambda1vr"
-		android:versionCode="27"
-		android:versionName="1.4.9" android:installLocation="auto" >
+		android:versionCode="28"
+		android:versionName="2.0.0" android:installLocation="auto" >
```

Comment:
I was planning to migrate the engine to the rebooted fork so I put v2, you can edit it if you are planning to release a new build